docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -0,0 +1,239 @@
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+  */
+////
+
+[[query-execution]]
+= Query Execution
+
+This section describes CQDs that are used to influence query execution.
+
+[[hbase-async-operation]]
+== HBASE_ASYNC_OPERATION
+
+[cols="25%h,75%"]
+|===
+| *Description*               | Allows index maintenance to be performed concurrently with base table operation. 
+| *Values*                    | *'ON'* Index maintenance is allowed. +
+*'OFF'* Index maintenance is not allowed. +
+ +
+The default value is *'ON'*.
+| *Usage*                     | HBase `put` operations are blocking. When the table has one or more indexes, 
+then the insert/update/delete (IUD) operation response time is improved by executing the index maintenance 
+operations concurrently with the base table operation: the put operations to these HBase tables are executed 
+in different threads.
+| *Production Usage*          | Yes.  It is 'ON' by default. This feature can be disabled by setting this CQD to 'OFF'.
+| *Impact*                    | IUD operations on tables with one or more indexes can become slower.
+| *Level*                     | Query.
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hbase-cache-blocks]]
+== HBASE_CACHE_BLOCKS
+
+[cols="25%h,75%"]
+|===
+| *Description*               | Influences HBase to retain the data blocks in memory after they are read. 
+| *Values*                    |
+*'ON'/'OFF'/'SYSTEM'*  +
+ +
+The default value is *'SYSTEM'*.
+| *Usage*                     | HBase maintains the block cache structure to retain the data blocks in memory after they are read.
+In LRU block cache configuration, the amount of block cache retained in memory is proportional to the amount of reserved maximum
+Java heap size of the region server. LRU Block Cache is the default in HBase. +
+ +
+The Trafodion Optimizer determines whether a sequential scan of the HBase table in a query would cause the full eviction of the
+data blocks cached earlier thereby impacting the performance of the random reads. The cache blocks option is turned off for the table
+in such a case. +
+ +
+Set the CQD <<hbase-region-server-max-heap-size,HBASE_REGION_SERVER_MAX_HEAP_SIZE>> value to reflect the amount of java heap size reserved for the region servers.
+This CQD is used by the Trafodion Optimizer to evaluate if the block cache to be turned off. 
```

Comment:
Suggested wordsmith: "This CQD is used by the Trafodion Optimizer to decide if the block cache should be turned off."